Occasional test failure: ERROR:test-mu-maildir.c:440:assert_matches_regexp: assertion failed

I sometimes (but not always) get this test failure when running the test suite:

```
Making all in tests
make[2]: Entering directory '/builddir/build/BUILD/mu-1.2-rc1/lib/tests'
make  all-am
make[3]: Entering directory '/builddir/build/BUILD/mu-1.2-rc1/lib/tests'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/builddir/build/BUILD/mu-1.2-rc1/lib/tests'
make[2]: Leaving directory '/builddir/build/BUILD/mu-1.2-rc1/lib/tests'
make[2]: Entering directory '/builddir/build/BUILD/mu-1.2-rc1/lib/parser'
TEST: test-tokenizer... (pid=11023)
  /tokens/basic:                                                       OK
  /tokens/specials:                                                    OK
  /tokens/ops:                                                         OK
  /tokens/escape:                                                      OK
  /tokens/to-string:                                                   OK
PASS: test-tokenizer
TEST: test-parser... (pid=11025)
  /parser/basic:                                                       OK
  /parser/complex:                                                     OK
  /parser/range:                                                       OK
  /parser/flatten:                                                     OK
PASS: test-parser
TEST: test-utils... (pid=11026)
  /utils/date-basic:                                                   OK
  /utils/date-ymwdhMs:                                                 OK
  /utils/size:                                                         OK
  /utils/flatten:                                                      OK
  /utils/clean:                                                        OK
  /utils/format:                                                       OK
PASS: test-utils
make[2]: Leaving directory '/builddir/build/BUILD/mu-1.2-rc1/lib/parser'
make[2]: Entering directory '/builddir/build/BUILD/mu-1.2-rc1/lib/tests'
make  all-am
make[3]: Entering directory '/builddir/build/BUILD/mu-1.2-rc1/lib/tests'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/builddir/build/BUILD/mu-1.2-rc1/lib/tests'
TEST: test-mu-util... (pid=11034)
  /mu-util/mu-util-dir-expand-00:                                      OK
  /mu-util/mu-util-dir-expand-01:                                      OK
  /mu-util/mu-util-guess-maildir-01:                                   OK
  /mu-util/mu-util-guess-maildir-02:                                   OK
  /mu-util/mu-util-check-dir-01:                                       OK
  /mu-util/mu-util-check-dir-02:                                       OK
  /mu-util/mu-util-check-dir-03:                                       OK
  /mu-util/mu-util-check-dir-04:                                       OK
  /mu-util/mu-util-get-dtype-with-lstat:                               OK
  /mu-util/mu-util-supports:                                           OK
  /mu-util/mu-util-program-in-path:                                    OK
PASS: test-mu-util
TEST: test-mu-str... (pid=11036)
  /mu-str/mu-str-size-01:                                              OK
  /mu-str/mu-str-size-02:                                              OK
  /mu-str/mu-str-prio-01:                                              OK
  /mu-str/mu-str-prio-02:                                              OK
  /mu-str/mu-str-display_contact:                                      OK
  /mu-str/mu-str-from-list:                                            OK
  /mu-str/mu-str-to-list:                                              OK
  /mu-str/mu-str-to-list-strip:                                        OK
  /mu-str/mu-str-parse-arglist:                                        OK
  /mu-str/mu-str-replace:                                              OK
  /mu-str/mu-str-esc-to-list:                                          OK
  /mu-str/mu_str_subject_normalize:                                    OK
  /mu-str/mu_str_remove_ctrl_in_place:                                 OK
PASS: test-mu-str
TEST: test-mu-maildir... (pid=11037)
  /mu-maildir/mu-maildir-mkdir-01:                                     OK
  /mu-maildir/mu-maildir-mkdir-02:                                     OK
  /mu-maildir/mu-maildir-mkdir-03:                                     OK
  /mu-maildir/mu-maildir-mkdir-04:                                     OK
  /mu-maildir/mu-maildir-mkdir-05:                                     OK
  /mu-maildir/mu-maildir-walk-01:                                      mkdir: cannot create directory '/tmp/mu-test-1001/test-a35e4dad': File exists
OK
  /mu-maildir/mu-maildir-walk:                                         mkdir: cannot create directory '/tmp/mu-test-1001/test-5955d863': File exists
OK
  /mu-maildir/mu-maildir-walk-with-noupdate:                           mkdir: cannot create directory '/tmp/mu-test-1001/test-a760d162': File exists
OK
  /mu-maildir/mu-maildir-get-new-path-new:                             **
ERROR:test-mu-maildir.c:440:assert_matches_regexp: assertion failed: (0)
FAIL
GTester: last random seed: R02Sf3062b46e8e42e48e4d4d0e1332844e0
make[2]: Leaving directory '/builddir/build/BUILD/mu-1.2-rc1/lib/tests'
/bin/sh: line 5: 11033 Terminated              gtester --verbose test-mu-util test-mu-str test-mu-maildir test-mu-msg-fields test-mu-msg test-mu-store test-mu-flags test-mu-container
make[2]: *** [Makefile:1009: test] Error 143
make[1]: *** [Makefile:1075: test] Error 2
make[1]: Leaving directory '/builddir/build/BUILD/mu-1.2-rc1/lib'
make: *** [Makefile:994: test] Error 2
```

--- a/lib/tests/test-mu-maildir.c
+++ b/lib/tests/test-mu-maildir.c
@@ -484,7 +484,7 @@ test_mu_maildir_get_new_path_new (void)
 		assert_matches_regexp (newbase,
 				       "\\d+\\."
 				       "[[:xdigit:]]{16}\\."
-				       "[[:alpha:]]+(:2,.*)?");
+				       "[[:alnum:]][[:alnum:]-]+(:2,.*)?");
 		g_free (newbase);
 		g_free(str);
 	}

